packages/components/src/components/input/input.scss

```diff
@@ -86,6 +87,9 @@
             -webkit-appearance: none;
             margin: 0;
         }
+        &[type=number] {
+            -moz-appearance: textfield;
```

Comment:
Adding `-moz-` should be unnecessary when you have postcss prefix set, no?

Reply:
@mahdipourziaei-binary The vendor prefixes are adding automatically. This one is an exception! because hiding number spinners [is different in chrome and firefox](https://stackoverflow.com/questions/3790935/can-i-hide-the-html5-number-input-s-spin-box), unfortunately. 
and this is not just a prefix addition. unless you have a better idea.

Reply:
@mahdiaryayi-fs Ah, I see. Alright then, thanks. 👍